lib/supportFeature.js

```diff
@@ -1,4 +1,22 @@
 const features = require('./features')
 
-module.exports = (feature, version) =>
-  features.some(({ name, versions }) => name === feature && versions.includes(version))
+function versionIndex (versionString) {
```

Comment:
This is really slow and could be optimised by building the relevant dictionaries
Might be important as this function is called often.
In fact we may even precompute all the supported feature for the current version.
However it looks like it should work, let's optimize it in another PR